src/structures/GroupDMChannel.js

```diff
@@ -125,18 +125,51 @@ class GroupDMChannel extends Channel {
   }
 
   /**
-   * Add a user to the DM
-   * @param {UserResolvable|string} accessTokenOrUser Access token or user resolvable
-   * @param {string} [nick] Permanent nickname to give the user (only available if a bot is creating the DM)
+   * Set a new name for this Group DM.
+   * @param {string} name New name for this Group DM
    * @returns {Promise<GroupDMChannel>}
    */
-  addUser(accessTokenOrUser, nick) {
-    const id = this.client.resolver.resolveUserID(accessTokenOrUser);
+  setName(name) {
```

Comment:
This method should follow the consistency of the rest of the API. There should be a `GroupDMChannel#edit` method and then `setName` should call it.